src/sections/Projects/Project-grid/projectGrid.style.js

```diff
@@ -275,6 +275,21 @@ export const ProjectWrapper = styled.div`
             }
         }
     }
+
+    
+    .project__card.eleven {
+        grid-column: 7/8;
+        grid-row: 4/6;
+        img {
+            width: 90px;
+            align-self: center;
+            margin-top: 16px
+        }
+        h5 {
+            margin-bottom: 25px;
+        }
```

Comment:
Can we use **flex** instead of **margin** to avoid **responsiveness** and **consistency** issues? There are some **media queries** used for the same in other cards. ( Refer to **tenth** card styles )
![image](https://github.com/layer5io/layer5/assets/29772969/927c8d83-d7fc-49b6-85da-6c920cc21bc8)
Here the spacing between the image and the title is inconsistent.

Reply:
Okay i will correct that